strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/ArtifactManagementService.java

```diff
@@ -141,12 +151,16 @@ public void releaseLock(URI pathUri)
 
     public void acquireLock(URI pathUri)
```

Comment:
@sbespalov , @fuss86 :

This method is the only one of importance. The rest is noise, due to the fact that I based this pull request on Sergey's last pull on my branch. I will rebase once I've merged #691.